front/src/views/Boxes/BoxesView.tsx

```diff
@@ -244,28 +243,27 @@ function Boxes() {
         Filter: SelectColumnFilter,
         filter: "includesOneOfMultipleStrings",
       },
+      {
+        Header: "Product Category",
+        accessor: "productCategory",
+        id: "productCategory",
+        Filter: SelectColumnFilter,
+        filter: "includesOneOfMultipleStrings",
+      },
     ],
     [],
   );
 
   // TODO: pass tag options to BoxesActionsAndTable
   return (
-    <>
-      <BreadcrumbNavigation items={[{ label: "Aid Inventory" }, { label: "Manage Boxes" }]} />
```

Comment:
@jlin95 probably a merge problem here, since Felipe added a heading and Breadcrumb here. Please re-add

Reply:
Hey I added the breadcrumb and navigation in an earlier PR but I figure I could also rebase. thanks!

Reply:
Never mind I added it back since it got removed from master.